### ISSUE TITLE
Customized pretrained models

### ISSUE THREAD

**Issue by kyleub** [mlfoundations/open_clip#527]
Hi, thanks for awesome repo. Is there any way to load a pre-trained model and change its layer config. For instance, I need to load ViT-B-16-plus-240, change the embed dim and after that I train it. Please note that I need to initialize the model's layer weights to the pre-trained model except for the layers which are changed.

**Comment by mitchellnw** [comment]
Maybe I misunderstand but I believe a pre-trained model will not work if you change the embed dim

**Comment by rwightman** [comment]
@kyleub changing the embed dim would change all of the layers so pretrained weights are irrelevant. You can certainly define a custom model with different embed dim but you'd have to train yourself.
